dist/secure-handlebars.js

```diff
@@ -8950,7 +8950,7 @@ Authors: Nera Liu <neraliu@yahoo-inc.com>
 var debug = require('debug')('cph');
 
 /* import the required package */
-var CustomizedContextParser = require('./customized-context-parser.js'),
+var ContextParser = require('./strong-context-parser.js'),
```

Comment:
i'd like to keep it first. that line will become
`var ContextParser = require('context-parser');`
if the changes in this PR are eventually adopted to the upstream.